Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -1550,6 +1550,18 @@
     <!-- XTXT: Statistics FAQ text -->
     <string name="statistics_faq_text">"Weitere Informationen finden Sie in den FAQ:"</string>
 
+    <!-- Seven Day Hospitalization Statistics Card -->
+    <!-- XHED: Title for seven day hospitalization stats -->
+    <string name="statistics_seven_day_hospitalization_card_title">"7-Tage-Inzidenz Hospitalisierung"</string>
+    <!-- XTXT: Hospitalizations per 100.000 inhabitants -->
+    <string name="statistics_seven_day_hospitalization_per_inhabitants_text">"COVID-19- Hospitalisierungen je 100.000 Einwohner"</string>
```

Comment:
is there a space between "-" and "Hospitalisierungen" because auf the line break? Otherwise, please take it out :)

Reply:
That's how the string were put in Figma but I believe the space shouldn't be there. I'll take it out :)

Reply:
Okay, this must have been because of the display, so that the texts are displayed with the correct line breaks. Thanks!